.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. -->
+
+
+## Use case
```

Comment:
Shall we remove this section, as use case should be clear from documentation and story?

Reply:
Not clearly sure, because the use case that developer imagines not always strictly equals his implementation.
But yes, we can remove that section.

I'll remove this section anyways - i's a bit excessive.